pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// ----------------------------
+// #SETTINGS
```

Comment:
Mind re-labelling this section and organizing this partial so that its similar to other pattern partials (https://github.com/edx/ux-pattern-library/blob/master/pattern-library%2Fsass%2Fpatterns%2F_lists.scss)? Specifically, this mixin should be under a "Utilities" area and any additional theming/configuration should be in a separate initial area called "Config".